app/src/main/res/values/dimens.xml

```diff
@@ -9,8 +9,9 @@
     <dimen name="news_picture_height">160dp</dimen>
```

Comment:
JFYI, It's not mandatory to create a `dimen` per each need. As per Material Design guidelines - a grid step size is 8dp, so each size should divide on 8 or at least 4 without a remainder. For example, you can't use 2/3/5/9/11/122dp but can use 4/8/12/16/120dp. You can also re-use your `dimen`. For example, if you see that you need the same size per margin/padding start and margin/padding end - you're welcome to call it something like padding_8/16/24/etc and use it whenever you need a padding/margin. This works with other shareable stuff as well. There's also a set of standard font sizes:

Type | Dimension
-- | --
Micro | 12 sp
Small | 14 sp
Medium | 18 sp
Large | 22 sp

But those are somewhat obsolete as we can customize them. Please, take a look at [this doc](https://material.io/design/typography/#type-scale) for more info.

When you try to name resources, try to start with the module/feature name and the finish with the component name - it would be simpler to group them this way.